components/national-team-profile.js

```diff
@@ -0,0 +1,343 @@
+const Label = dynamic(() => import('components/label'));
+const Button = dynamic(() => import('components/button'));
+const Container = dynamic(() => import('components/container'));
+const Required = dynamic(() => import('components/required'));
+const InlineError = dynamic(() =>
+  import('components/errors').then(({ InlineError }) => InlineError)
+);
+
+import { api } from 'modules/api';
+import { rem } from 'styles/theme';
+
+const NATIONAL_TEAMS = ['England', 'Scotland', 'Wales'];
+const POSITIONS = ['Keeper/Chaser', 'Beater', 'Seeker'];
+
+const NationalTeamFormSchema = object().shape({
+  first_team: string()
+    .nullable()
+    .required('Please select your first choice National Team.'),
+  second_team: string().nullable(),
+  third_team: string().nullable(),
+  position: string().required('Please select your primary position.'),
+  playstyle: string().required(
+    'Please enter a short description of your playstyle.'
+  ),
+  years: number().required(
+    'Please estimate how many years of active quidditch experience you have.'
+  ),
+  experience: string().required('Please summarise your quidditch experience.'),
+});
+
+const handleFormSubmit = async (values, setServerError, setServerSuccess) => {
+  try {
+    setServerError(null);
+    setServerSuccess(null);
+
+    // TODO: Update and test api endpoint for this form.
+    await api.post('/contact/national', values);
+    setServerSuccess(true);
+  } catch (err) {
+    setServerError(err?.response?.data?.error?.message);
+  }
+};
+
+// TODO: Test this to recieving a data structure provided by the API.
+const NationalTeamProfileForm = ({ profile }) => {
+  const [serverError, setServerError] = useState(null);
+  const [serverSuccess, setServerSuccess] = useState(null);
+
+  const { register, handleSubmit, errors, watch, formState } = useForm({
+    mode: 'onBlur',
+    resolver: yupResolver(NationalTeamFormSchema),
+    defaultValues: {
+      national_team_interest: profile.national_team_interest,
+      first_team: profile.first_team,
+      second_team: profile.second_team,
+      third_team: profile.third_team,
+      position: profile.position,
+      playstyle: profile.playstyle,
+      years: profile.years,
+      experience: profile.experience,
+    },
+  });
+
+  const { isSubmitting } = formState;
+
+  const watchInterest = watch('national_team_interest');
+  const variant = 'light';
+
+  // TODO: Not sure exactly how the forms work regarding linking back to the user. But that needs to be handled here.
+  return (
+    <PrismicWrapper variant={variant}>
+      <Heading as="h1" fontFamily="body" textAlign="center">
+        National Team Profile
+      </Heading>
+      <p>
+        Welcome to <b>your</b> National Team Profile. Use the form below to get
+        on the radar of the best Managers and Coaches in the UK, and register
+        your interest in playing for one of QuidditchUKs National Teams.
```

Comment:
I'm fairly sure we never use possessive with QUK, so we can get around this like so
```suggestion
        your interest in playing for one of the QuidditchUK National Teams.
```